packages/react-storage/src/components/StorageBrowser/storageBrowser/types.ts

```diff
@@ -0,0 +1,149 @@
+import {
+  ActionConfigs,
+  DefaultActionConfigs,
+  DefaultActionKeys,
+} from '../actions/configs';
+import { createUseAction } from '../actions/createUseAction';
+import { StorageBrowserElements } from '../context/elements';
+import { Config } from '../createProvider';
+
+interface CreateStorageBrowserInput<
+  DefinedActionConfigs extends ActionConfigs,
+> {
```

Comment:
Prefer not to break the generic conventions that already exist in the code base, e.g.
```
interface CreateStorageBrowserInput<
  T extends ActionConfigs,
> { 
```